MovieHacker/TableViewController.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  ViewController.swift
+//  MovieHacker
+//
+//  Created by Jan Dědeček on 07/08/2017.
+//  Copyright © 2017 Jan Dědeček. All rights reserved.
+//
+
+import UIKit
+
+let apiKey = ""
+
+class TableViewController: UITableViewController {
+
+  var task: URLSessionDataTask?
+
+  var model = [Movie]()
+
+  let imageCache = NSCache<NSString, UIImage>()
```

Comment:
conceptual note: how about commenting the code like a DEMO not like a production code?
for instance, here i'd leave a short comment that reads like: "A thread-safe NSDictionary-ish data structure"